SIT.Manager.Avalonia/ViewModels/InstallPageViewModel.cs

```diff
@@ -42,9 +33,45 @@ public partial class InstallPageViewModel : ObservableRecipient,
     [ObservableProperty]
     private ReadOnlyCollection<InstallStep> _installationSteps;
 
-    public InstallPageViewModel()
+    public InstallPageViewModel(ILocalizationService localizationService)
     {
+        _localizationService = localizationService;
+        _localizationService.LocalizationChanged += (o, e) =>
+        {
+
+            _sitInstallSteps =
+            [
+                new(typeof(SelectView), _localizationService.TranslateSource("InstallPageViewModelSelectText")),
+                new(typeof(ConfigureSitView), _localizationService.TranslateSource("InstallPageViewModelConfigureText")),
+                new(typeof(PatchView), _localizationService.TranslateSource("InstallPageViewModelPatchText")),
+                new(typeof(InstallView), _localizationService.TranslateSource("InstallPageViewModelInstallText")),
+                new(typeof(CompleteView), _localizationService.TranslateSource("InstallPageViewModelCompleteText"))
+            ];
+            _serverInstallSteps =
+            [
+                new(typeof(SelectView), _localizationService.TranslateSource("InstallPageViewModelSelectText")),
+                new(typeof(ConfigureServerView), _localizationService.TranslateSource("InstallPageViewModelConfigureText")),
+                new(typeof(InstallView), _localizationService.TranslateSource("InstallPageViewModelInstallText")),
+                new(typeof(CompleteView), _localizationService.TranslateSource("InstallPageViewModelCompleteText"))
+            ];
+            InstallationSteps = _sitInstallSteps.AsReadOnly();
+        };
         WeakReferenceMessenger.Default.RegisterAll(this);
+        _sitInstallSteps =
+        [
+            new(typeof(SelectView), _localizationService.TranslateSource("InstallPageViewModelSelectText")),
+            new(typeof(ConfigureSitView), _localizationService.TranslateSource("InstallPageViewModelConfigureText")),
+            new(typeof(PatchView), _localizationService.TranslateSource("InstallPageViewModelPatchText")),
+            new(typeof(InstallView), _localizationService.TranslateSource("InstallPageViewModelInstallText")),
+            new(typeof(CompleteView), _localizationService.TranslateSource("InstallPageViewModelCompleteText"))
+        ];
+        _serverInstallSteps = 
+        [
+            new(typeof(SelectView), _localizationService.TranslateSource("InstallPageViewModelSelectText")),
+            new(typeof(ConfigureServerView), _localizationService.TranslateSource("InstallPageViewModelConfigureText")),
+            new(typeof(InstallView), _localizationService.TranslateSource("InstallPageViewModelInstallText")),
+            new(typeof(CompleteView), _localizationService.TranslateSource("InstallPageViewModelCompleteText"))
+        ];
```

Comment:
Just a note so you guys know I think this is just gonna cause so much more hassle in the future than it's worth (and you can kind of see already if you open the app, try a server install, and  look closely at the stepper and you'll probably see part of what I mean) with dynamically changing the Stepper values like this, so at some point I'll change probably them to just be in XAML.